setup.py

```diff
@@ -18,58 +18,67 @@ def read(*names, **kwargs):
     ).read()
 
 
+package_dir = 'src'
+packages = find_packages(package_dir)
+root_packages = [
+    package
+    for package in packages
+    if "." not in package
+]
+
+assert len(root_packages) == 1
+package = root_packages[0]
+
 version_ns = {}  # type: ignore
-execfile(pjoin(repo_root, 'src', 'pymedphys', '_version.py'), version_ns)
+version_filepath = glob(
+    pjoin(root, package_dir, package, '_version.py'))[0]
+execfile(version_filepath, version_ns)
 
 version = version_ns['__version__']
 
 
 setup(
-    name="pymedphys",
+    name=package,
     version=version,
-    author="Simon Biggs",
-    author_email="me@simonbiggs.net",
+    author="PyMedPhys Contributors",
+    author_email="developers@pymedphys.com",
     description='Medical Physics python modules',
     long_description=read('README.rst'),
     long_description_content_type='text/x-rst',
     classifiers=[
         'Development Status :: 4 - Beta',
         'License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)',
-        'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Topic :: Scientific/Engineering :: Medical Science Apps.',
         'Topic :: Scientific/Engineering :: Physics',
         'Intended Audience :: Science/Research',
         'Intended Audience :: Healthcare Industry'
     ],
-    packages=find_packages('src'),
-    package_dir={'': 'src'},
-    py_modules=[splitext(basename(path))[0] for path in glob('src/*.py')],
+    packages=packages,
+    package_dir={'': package_dir},
     include_package_data=True,
     package_data={'pymedphys': []},
     entry_points={
         'console_scripts': [
             'pymedphys=pymedphys.cli.main:pymedphys_cli'
         ],
     },
-    license='AGPLv3+',
+    license='AGPL-3.0-or-later',
```

Comment:
Does the '+' in the old version signify "or later" or "with added terms"?

Reply:
In symbolises "or later". This isn't a license change, just that there has been a consensus that the + was not sufficiently explicit within the community.